app/models/course.rb

```diff
@@ -323,15 +314,69 @@ class Course < ActiveHash::Base
             'You are an intermediate-level developer',
             'You might be familiar with version control and basic web architecture'
           ]
+        }
+      ]
+    }, {
+      id: 8,
```

Comment:
I'd use the i18n strategy instead, even if it's just for the course description, since the responsibility of this model is to solely be a course index, and having one of its elements in a different language will give the idea of an existing course in Portuguese, but it isn't really there. 

I think localization would be a really nice-to-have, but definitely requires a lot of work and coordination, and overall, it requires time. It would be awesome if we could build a team willing to work specifically on localization, so it can be done more quickly. 

I think we can discuss this topic in the issues section. And of course, count on me for any localization work needed :smile: